lang/test/org/partiql/lang/eval/visitors/SelectStarTransformTests.kt

```diff
@@ -0,0 +1,157 @@
+package org.partiql.lang.eval.visitors
+
+import junitparams.Parameters
+import org.junit.Test
+
+class SelectStarTransformTests : VisitorTransformTestBase() {
+    private val xformer = PipelinedTransformer(GroupByItemAliasTransform(), SelectStarTransform())
+
+    @Test
+    @Parameters
+    fun tests(tc: XformTestCase) = runTestForIdempotentRewriter(tc, xformer)
+
+    fun parametersForTests() = listOf(
+        XformTestCase(
+            """
+                SELECT * 
+                FROM foo AS f
+            """,
+            """
+                SELECT "f".* 
+                FROM foo AS f
+            """
+        ),
+        XformTestCase(
+            """
+                SELECT DISTINCT * 
+                FROM foo AS f
+            """,
+            """
+                SELECT DISTINCT 
+                    "f".* 
+                FROM foo AS f
+            """
+        ),
+        XformTestCase(
+            """
+                SELECT * 
+                FROM foo AS f AT idx
+            """,
+            """
+                SELECT 
+                    "f".*, 
+                    "idx" AS idx 
+                FROM foo AS f AT idx
+            """
+        ),
+        XformTestCase(
+            """
+                SELECT * 
+                FROM foo AS f AT idx BY addr
+            """,
+            """
+                SELECT 
+                    "f".*, 
+                    "idx" AS idx, 
+                    "addr" as addr 
```

Comment:
Does the assertion ignore case? `AS` on line 55 vs `as` line 56

Reply:
This is parsed and the resulting trees are compared, so yes, in this case they do ignore case.